ios/FirebaseContinue/Example/FirebaseContinue/FCNViewController.m

```diff
@@ -16,22 +16,41 @@
 
 #import "FCNViewController.h"
 
+@import FirebaseContinue;
+
+/**
+ * Important Note: This is not a working example of using Firebase Continue.
+ *
+ * The Firebase Continue for iOS library is planned to eventually be available via Cocoapods
+ * (rather than being required to manually copy-paste its source into your project), so this
+ * Cocoapods-provided development environment lays the groundwork for that.
+ *
+ * For an actual, working sample of Firebase Continue, see the samples/ios subdirectory
+ * from the root of the repository at:
+ * https://github.com/firebase/firebase-continue/tree/master/samples/ios
+ */
 @interface FCNViewController ()
 
 @end
 
 @implementation FCNViewController
 
-- (void)viewDidLoad
+- (void)viewDidAppear:(BOOL)animated
 {
-    [super viewDidLoad];
-	// Do any additional setup after loading the view, typically from a nib.
-}
+  [super viewDidAppear:animated];
 
-- (void)didReceiveMemoryWarning
-{
-    [super didReceiveMemoryWarning];
-    // Dispose of any resources that can be recreated.
+  // Attempt to use Firebase Continue here. This should fail for any number of reasons,
+  // including that Firebase has not been set up for this app.
+  [FCNContinue broadcastToContinueActivityWithUrl: @"http://some.url.com"
```

Comment:
Done (changed this code here to match the code in the sample usage guide -- so it's not really valid anyways, but since this code isn't even meant to be run, it makes sense).